### DISCUSSION TITLE
How to add BreakpointSplitView when embedding in a website

### DISCUSSION THREAD

**Discussion by xiaonui** [GMOD/jbrowse-components#4176] [Q&A]
Hi, I have successfully embedded JBrowse into my website. I would like to further display the breakpoints for BND variants. But when the view type is set to BreakpointSplitView, it reports an error saying that this view type could not be found. How can I implement this functionality? Thanks in advance!

**Comment by cmdcolin** [comment]
do you have a screenshot of the error you are seeing? are you using the @jbrowse/react-linear-genome-view method for embedding? you might need to use @jbrowse/react-app which is the method for embedding the "full web app" with the multiple view types in it

**Reply by cmdcolin** [reply]
see https://jbrowse.org/jb2/docs/embedded_components/ for react-app embedding examples
